sklearn/model_selection/tests/test_split.py

```diff
@@ -1622,6 +1623,113 @@ def test_random_state_shuffle_false(Klass):
         Klass(3, shuffle=False, random_state=0)
 
 
+def test_group_time_series_fail_groups_are_none():
+    """
+    The GroupTimeSeriesSplit with no group should raise an Error
+    """
```

Comment:
I think we do not use docstrings for tests.
```suggestion
    # The GroupTimeSeriesSplit with no group should raise an Error
```

Reply:
Done